spec/models/beer_spec.rb

```diff
@@ -0,0 +1,60 @@
+require 'rails_helper'
+
+RSpec.describe Beer, type: :model do
+  let(:beer) { Beer.create(name: "Titan", 
+                           state: true,
+                           description: "aggressively hopped ipa", 
+                           price: 350) 
+              }
```

Comment:
`end` at 8, 14 is not aligned with `let(:beer) { Beer.create(name: "Titan", ` at 4, 2